cheroot/server.py

```diff
@@ -1175,6 +1175,12 @@ def send_headers(self):
                 if not self.close_connection:
                     self.outheaders.append((b'Connection', b'Keep-Alive'))
 
+        if dict(self.outheaders).get(b'Connection', b'') == b'Keep-Alive':
+            self.outheaders.append((
+                b'Keep-Alive',
+                'timeout={}'.format(self.server.timeout).encode('ISO-8859-1'),
```

Comment:
Since we still support py2, let's make sure that we always encode unicode text explicitly
```suggestion
                u'timeout={}'.format(self.server.timeout).encode('ISO-8859-1'),
```